tcat-app/inputparams.py

```diff
@@ -1,39 +1,57 @@
 # array structure [control_name , datatype , lookup_name , speaking_name , default_value , accepted_values]
 
 constellation_mission_params = {
-    ('mission_configuration', 'Mission Configuration'): [
-        ['select', 'text', 'propulsion_type', 'Propulsion type', 'electrical',
-         ['electrical', 'water', 'mono-propellant', 'solid', 'bi-propellant']],
+    ('scenario_metadata', 'Scenario Metadata'): [
         ['checkbox', '', 'verbose', 'Debug (display figures)', False, [True, False]],
-        ['input', 'datetime-local', 'starting_epoch', 'Starting Epoch', '2025-01-01T12:00',
-         ['2000-01-01T00:00', '2100-01-01T00:00']],
-        ['input', 'number', 'mission_cash_limitor', 'Mission Cash Limitor', 0.01, [0.0, 1.0]],
+        ['input', 'datetime-local', 'starting_epoch', 'Starting Epoch', '2025-01-01T12:00', ['2000-01-01T00:00', '2100-01-01T00:00']],
+        ['input', 'number', 'tradeoff_mission_price_vs_duration', 'Tradeoff Mission Price vs Duration', 0.01, [0.0, 1.0]],
     ],
-    ('constellation_configuration', 'Constellation Configuration'): [
-        ['input', 'text', 'constellation_name', 'Constellation Name', 'My Constellation', '^[A-Za-z0-9 ]+$'],
+    ('constellation', 'Constellation'): [
+        ['input', 'text', 'constellation_name', 'Name', 'My Constellation', '^[A-Za-z0-9 ]+$'],
         ['input', 'number', 'sat_mass', 'Satellite Mass (kg)', 147, [0.1, 99999.9]],
         ['input', 'number', 'sat_volume', 'Satellite Volume (m^3)', 1.3, [0.001, 99999.9]],
         ['input', 'number', 'n_planes', 'Number of orbital planes', 3, [1, 100]],
         ['input', 'number', 'n_sats_per_plane', 'Number of satellite per orbital plane', 2, [1, 1000]],
         ['input', 'number', 'plane_distribution_angle', 'Distribution angle of orbital planes (deg)', 180, [180, 360]],
     ],
-    ('launch_vehicle_configuration', 'Launch Vehicle Configuration'): [
-        ['select', 'text', 'launcher', 'Launch vehicle name and variant', 'Soyuz_2.1a_Fregat',
-         ['Ariane_62', 'Ariane_64', 'Soyuz_2.1a_Fregat', 'Soyuz_2.1b_Fregat', 'Soyuz']],
+    ('launcher', 'Launcher'): [
+        ['checkbox', '', 'launcher_use_database', 'Use database', True, [True, False]],
+        ['select', 'text', 'launcher', 'Launch vehicle name and variant', 'Soyuz_2.1a_Fregat', ['Ariane_62', 'Ariane_64', 'Soyuz_2.1a_Fregat', 'Soyuz_2.1b_Fregat', 'Soyuz']],
         ['select', 'text', 'launch_site', 'Launch Site', 'Baikonur', ['Korou', 'Baikonur']],
         ['select', 'text', 'orbit_type', 'Orbit Type', 'LEO', ['LEO', 'SSO', 'LPEO', 'Polar', 'MTO', 'GTO', 'GTO+']],
         ['input', 'number', 'dispenser_tech_level', 'Dispenser technological level', 1.1, [0.1, 1.9]],
```

Comment:
dispenser tech level is not required anymore

Reply:
strange it appears here in black but is not in the file when I open without the review, so apparently it's already removed